### DISCUSSION TITLE
Force refresh of HA devices

### DISCUSSION THREAD

**Discussion by ghost** [JoDehli/PyLoxone#312] [Q&A] [answered]
After initializing the integration, I had 90 new devices in Home-Assistant. Later, I added a few more devices in Loxone and ensured that the user logged into the PyLoxone integration had access to these new Loxone devices. I assumed pressing the reload button would add the new Loxone devices in HA, but it didn't; the count remained at 90 devices. Subsequently, I tried removing a device's access for the given user, but this also did not reflect any change in HA; the device count persisted at 90.

I confirmed via `http://<Miniserver-ip:port>/data/LoxAPP3.json` with the user logged into the integration, and here the devices were correctly updated.

I'm wondering if I might be reloading the integration incorrectly. Could someone describe the correct procedure to reload the devices/entities from Loxone into HA using the PyLoxone integration?

**Comment by OndraaasCZ** [comment, marked as answer]
Restart Home Assistant.
